client/src/config/mapbox.js

```diff
@@ -10,5 +10,19 @@ const tileOptions = {
   accessToken: import.meta.env.VITE_MAPBOX_API_KEY,
 };
 
+// TODO: Check attribution: https://docs.mapbox.com/help/getting-started/attribution/
```

Comment:
Ein Zoom auf den Satellite Layer führt zu nachfolgendem Error

Uncaught TypeError: map is null
    _updateLevels GridLayer.js:389
    _setView GridLayer.js:566
    _animateZoom GridLayer.js:530
    fire Events.js:190
    _animateZoom Map.js:1689
    _tryAnimatedZoom Map.js:1667
    requestAnimFrame Util.js:228
    _tryAnimatedZoom Map.js:1664
    setView Map.js:195
    setZoomAround Map.js:249
    _performZoom Map.ScrollWheelZoom.js:83
    setTimeout handler*_onWheelScroll Map.ScrollWheelZoom.js:57
    handler DomEvent.js:92
    addOne DomEvent.js:107
    on DomEvent.js:34
    addHooks Map.ScrollWheelZoom.js:33
    enable Handler.js:23
    _handleScroll leaflet-gesture-handling.js:247
    handler DomEvent.js:92
    addOne DomEvent.js:107
    on DomEvent.js:34
    addHooks leaflet-gesture-handling.js:32
    enable Handler.js:23
    addHandler Map.js:744
    init Class.js:106
    callInitHooks Class.js:78
    initialize Map.js:156
    NewClass Class.js:22
    createMap Map.js:1728
    setup FlightMap.vue:107

Reply:
fixed